com.unity.render-pipelines.high-definition/Documentation~/Decal-Projector.md

```diff
@@ -35,6 +35,7 @@ Using the Inspector allows you to change all of the Decal Projector properties,
 | ----------------------- | ------------------------------------------------------------ |
 | **Size**                | The 3D size of the projector influence box, and thus the decal. The projector scales the decal to match the **X** and **Z** components of the **Size**. The Decal Projector component projects decals along the local y-axis. |
 | **Material**            | The decal Material to project. The decal Material must use a HDRP/Decal Shader. |
+| **Decal Layer Mask**    | Materials with a matching Decal Layer Mask will be affected by this decal. |
```

Comment:
The layer mask that specifies the Materials to project its decal onto. Any Material that uses a matching Decal Layer Mask receives the decal.